src/webots/gui/WbDragTransformEvent.cpp

```diff
@@ -173,8 +173,8 @@ WbDragTranslateAlongAxisEvent::WbDragTranslateAlongAxisEvent(const QPoint &initi
   mTextOverlay->applyChangesToWren();
 
   WbMatrix4 matrix(mSelectedTransform->matrix());
-  const WbVector3 absoluteScale = matrix.scale();
-  matrix.scale(1.0f / absoluteScale.x(), 1.0f / absoluteScale.y(), 1.0f / absoluteScale.z());
+  WbVector3 scale = mSelectedTransform->scale();
```

Comment:
Can't we use `const` here?

Reply:
I did it originally but Cpp-check was complaining...

Reply:
Ok I misunderstood, in facts it was asking for a const reference, as you suggested.